product-mini/platforms/linux/CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ set_target_properties (iwasm PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
 install (TARGETS iwasm DESTINATION bin)
 
-target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} -lm -ldl -lpthread)
+target_link_libraries (iwasm vmlib ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS} ${TENSORFLOW_LIB} -lm -ldl -lpthread)
 
 add_library (libiwasm SHARED ${WAMR_RUNTIME_LIB_SOURCE})
```

Comment:
Don't we need to add "$NN_LIBS"(a.k.a ${TENSORFLOW_LIB})  for libiwasm as well?

Reply:
you are right! https://github.com/bytecodealliance/wasm-micro-runtime/pull/2308/commits/25f593a342d9e004635f6ccd303bfbcaaa90509b